src/definitions.ts

```diff
@@ -194,7 +208,12 @@ export interface AuthorizationCode {
    * Jwt
    * @description A JSON web token
    */
-  jwt: string;
+  jwt?: string;
+  /**
+   * Access Token
+   * @description An access token
+   */
+  accessToken?: string;
```

Comment:
this is not suppose to be here, ad we only touch google provider

Reply:
If you could help with the type magic, that would be amazing. I tried and I failed